deploy/helm/opa-operator/templates/deployment.yaml

```diff
@@ -43,6 +44,11 @@ spec:
           env:
             - name: OPA_BUNDLE_BUILDER_CLUSTERROLE
               value: {{ include "operator.fullname" . }}-opa-bundle-builder-clusterrole
+            - name: OPERATOR_IMAGE
+              # Tilt can use recognize annotations as image paths, but not env variables
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.annotations['image']
```

Comment:
Move into templating

Reply:
Done in https://github.com/stackabletech/operator-templating/pull/294